### DISCUSSION TITLE
Unable to build integration project

### DISCUSSION THREAD

**Discussion by Joeriubink** [openTCS/opentcs#111] [Q&A] [answered]
Operating system: Windows 10
JDK: 13

java -version
openjdk version "13.0.2" 2020-01-14
OpenJDK Runtime Environment AdoptOpenJDK (build 13.0.2+8)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 13.0.2+8, mixed mode, sharing)

javac -version
javac 13.0.2

F:\OpenTCS\openTCS-Integration-Test>gradlew build
> Task :openTCS-Test-Common:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':openTCS-Test-Common:compileJava'.
> error: release version 13 not supported

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 1s
5 actionable tasks: 1 executed, 4 up-to-date

**Comment by swltr** [comment, marked as answer]
@Joeriubink I'd assume that Gradle uses some other installed JDK here. That's actually the only reason I see why it should say "release version 13 not supported".

**Reply by Joeriubink** [reply]
Yes, that was the issue. I had a JAVA_HOME environment variable set to another JDK.
